test/plugin/scenarios/rocketmq-scenario/support-version.list

```diff
@@ -17,3 +17,8 @@
 # lists your version here (Contains only the last version number of each minor version.)
 
 4.9.4
+4.9.3
+4.9.2
+4.9.1
+4.9.0
+4.8.0
```

Comment:
```suggestion
4.9.3
4.8.0
```

Minor version means the 2nd part of the version. Read this, https://semver.org/#:~:text=A%20normal%20version%20number%20MUST,0%20%2D%3E%201.11.0.

And supported-list doc indicates, 4.x RocketMQ supported. From [here](https://rocketmq.apache.org/download) I can see a lot of versions of 4.x

Could you confirm whether we only support 4.8+ or really cover all 4.x?
We should update the document accordingly.

Reply:
Sure, we just need our test scenarios to match the announced versions. Otherwise, we may face confusions in the future.